apps/modernization-ui/src/apps/patient/data/sexAndBirth/asSex.ts

```diff
@@ -1,6 +1,6 @@
 import { asValue } from 'options';
-import { Sex } from './api';
-import { SexEntry } from './entry';
+import { Sex } from '../api';
+import { SexEntry } from '../entry';
```

Comment:
question (non-blocking): Should the types be moved down a folder also?

Reply:
Maybe. That would get rid of `api` and `entry`